agent/command/results_native.go

```diff
@@ -119,5 +123,41 @@ func (c *attachResults) Execute(ctx context.Context,
 		return errors.Wrapf(err, "reading report file '%s'", reportFileLoc)
 	}
 
+	if err := c.sendTestLogs(ctx, conf, logger, comm, &nativeResults); err != nil {
+		return errors.Wrap(err, "sending test logs")
+	}
+
 	return sendTestResults(ctx, comm, logger, conf, nativeResults.convertToService())
 }
```

Comment:
We have a sendTestLogsAndResults function. Can that be adapted to also work for raw logs instead of breaking it up like this?

Reply:
Good idea! I cleaned it up so that it will first create the test logs and then call `sendTestLogsAndResults`.